instrumentation/opentelemetry-instrumentation-psycopg2/tests/test_psycopg2_integration.py

```diff
@@ -208,6 +208,7 @@ def test_instrument_connection_with_instrument(self):
         self.assertEqual(len(spans_list), 0)
 
         Psycopg2Instrumentor().instrument()
+        cnx = psycopg2.connect(database="test")
```

Comment:
We seem to need this line added to pass because `cnx` needs to be created after the call to `instrument()`. The `self.assertEqual(len(spans_list), 1)` further down fails without it on my local, with or without the call to `instrument_connection()` immediately after. This test is now more like how the [docs describe using instrument()](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/psycopg2/psycopg2.html#id1) but it's weird that the test was passing before. 🤔